### DISCUSSION TITLE
Why window switcher not showing border at the selected window ?

### DISCUSSION THREAD

**Discussion by LAGGOUNE-Walid** [elementary/gala#1371] [Q&A] [answered]
When i change window using the keyboard shortcut ALT+TAB i could not know which position (window) i am scrolling to. its just showing the application name at the bottom .
![Screenshot from 2021-12-31 16 07 10](https://user-images.githubusercontent.com/16167585/147829966-5680bcb7-dd26-4c8e-a07b-703eb5782ec1.png)

I think and i remember it was something like this
 
![9uF2C](https://user-images.githubusercontent.com/16167585/147830000-62b3b6ba-c715-458e-8918-b40337afab5e.jpg)

**Comment by cassidyjames** [comment, marked as answer]
The window switcher uses the currently-selected accent color from System Settings. It looks like you're using a third-party stylesheet, which is not supported on elementary OS. Your stylesheet doesn't seem to be exporting the accent colors as the OS requires.

![Screenshot from 2022-01-03 19 31 06](https://user-images.githubusercontent.com/611168/148001861-bdebe5c4-ed56-4f54-bf7c-b13ff85c88c8.png)
